### PR TITLE
fixes nil address case

### PR DESCRIPTION
**Pull request by amprokop**

### CODE REVIEW

File: app/mappers/address_mapper.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module AddressMapper
-  def get_address_from_bgs_address(bgs_address = {})
+  def get_address_from_bgs_address()
```
Comment (kierachell) [inline app/mappers/address_mapper.rb]: Wouldn't this set the number of expected arguments to be 0 so it will error if any are passed.?

Comment (amprokop) [inline app/mappers/address_mapper.rb]: lol

Comment (amprokop) [inline app/mappers/address_mapper.rb]: yes, shit, sorry

**Comment by amprokop** [comment]
@kierachell — totally right, fat finger. fixed!

**Pull request merged**